automatic/minikube/update.ps1

```diff
@@ -15,30 +15,32 @@ function global:au_SearchReplace {
     ".\legal\VERIFICATION.txt" = @{
       "(?i)(^\s*location on\:?\s*)\<.*\>" = "`${1}<$($Latest.ReleaseURL)>"
       "(?i)(^\s*software.*)\<.*\>"        = "`${1}<$($Latest.URL64)>"
-      "(?i)(^\s*checksum\s*type\:).*"     = "`${1} $($Latest.ChecksumType32)"
+      "(?i)(^\s*checksum\s*type\:).*"     = "`${1} $($Latest.ChecksumType64)"
       "(?i)(^\s*checksum\:).*"            = "`${1} $($Latest.Checksum64)"
     }
 
     "$($Latest.PackageName).nuspec" = @{
-      "(\<releaseNotes\>).*?(\</releaseNotes\>)" = "`${1}$($Latest.ReleaseNotes)`${2}"
+      "(\<copyright\>.*)\d{4}(.*?\</copyright\>)" = "`${1}$($Latest.ReleaseYear)`${2}"
```

Comment:
Can you guarantee that this will always match the copyright specified by the Minikube authors?

If not, then we should not update this section automatically.